<launch>

  <node pkg="four_wheel_drive_independent_steering" name="local_planner" type="holonomic_dwa">
    <remap from="/fwdis/local_goal" to="/move_base_simple/goal"/>
    <param name="MAX_VELOCITY" value="2.0"/>
    <param name="MAX_ACCELERATION" value="1.0"/>
    <param name="MAX_ANGULAR_VELOCITY" value="4.0"/>
    <param name="MAX_ANGULAR_ACCELERATION" value="4.0"/>
    <param name="VELOCITY_RESOLUTION" value="0.10"/>
    <param name="ANGULAR_VELOCITY_RESOLUTION" value="0.10"/>
    <param name="SIMULATE_TIME" value="2.0"/>
    <param name="ROBOT_RADIUS" value="0.75"/>
    <param name="GOAL_XY_TOLERANCE" value="0.15"/>
    <param name="GOAL_YAW_TOLERANCE" value="0.15"/>
    <param name="ALPHA" value="1.0"/>
    <param name="BETA" value="1.0"/>
    <param name="GAMMA" value="1.0"/>
  </node>

</launch>
